bot/cogs/core/paginator.py

```diff
@@ -82,6 +84,11 @@ async def paginate_embed(
             elif emoji == RIGHT_ARROW:
                 if page < page_count - 1:
                     page += 1
+            elif emoji == SKIP_TO_LAST:
+                page += page_count - 1
```

Comment:
Shouldn't this be `page = page_count - 1`

Reply:
well it works with both `page = page_count - 1` and `page += page_count - 1`, I can make changes if you want the former

Reply:
> I think it only works with the latter if the page count is 0 though... so I will make the edit

Exactly

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/272/commits/8495b94bd486adf8c165693c59fcac5778ab34b0